source/NVDAObjects/UIA/__init__.py

```diff
@@ -2302,20 +2321,11 @@ def event_UIA_layoutInvalidated(self):
 class SuggestionListItem(UIA):
 	"""Recent Windows releases use suggestions lists for various things, including Start menu suggestions, Store, Settings app and so on.
 	Unlike suggestions list class, top suggestion is automatically selected.
+	Note that support for reporting the selection is now handled generically on the base NVDAObject.
 	"""
 
 	role = controlTypes.Role.LISTITEM
 
-	def event_UIA_elementSelected(self):

```

Comment:
same question here re backwards compatibility

Reply:
Again moved to the base.